website/server/controllers/api-v3/chat.js

```diff
@@ -101,6 +111,22 @@ api.postChat = {
     let lastClientMsg = req.query.previousMsg;
     chatUpdated = lastClientMsg && group.chat && group.chat[0] && group.chat[0].id !== lastClientMsg ? true : false;
 
+    // Check for spam by seeing if the current user has posted too many messages in a short window
```

Comment:
I don't think this comment is necessary. I think it would probably be better to pull this logic into an instance method on the group model.